### PR TITLE
update getting started doc

### PR DESCRIPTION
**Pull request by songjiayang**
1. change single-process-config-blocks.yaml to single-process-config-blocks-local.yaml
2. add specific tag for cortex docker image

Signed-off-by: songjiayang <songjiayang1@gmail.com>

### CODE REVIEW

**Comment by alanprot** [comment]
Humm.. i think changing to local make sense.

But I'm not sure about if we should pin a version as we will need to remember on all release to update this.

**Comment by yeya24** [comment]
> But I'm not sure about if we should pin a version as we will need to remember on all release to update this.

Yeah for getting started, using the latest image is better.

**Comment by songjiayang** [comment]
using latest version for default is better, but now when pull image with `latest` tag , it will raise a error:

```
docker run -d --name=cortex2 --network=cortex \
>     -v $(pwd)/docs/configuration/single-process-config-blocks-local.yaml:/etc/single-process-config-blocks-local.yaml \
>     -p 9002:9009 \
>     quay.io/cortexproject/cortex \
>     -config.file=/etc/single-process-config-blocks-local.yaml \
>     -ring.store=consul \
>     -consul.hostname=consul:8500 \
>     -distributor.replication-factor=3
Unable to find image 'quay.io/cortexproject/cortex:latest' locally
docker: Error response from daemon: manifest for quay.io/cortexproject/cortex:latest not found: manifest unknown: manifest unknown.
```

**Comment by yeya24** [comment]
Umm, yeah I don't see we have a image tagged with `latest` on quay... 
So probably the doc doesn't work from the beginning.

**Comment by friedrichg** [comment]
> Next we're going to show how you can run a scale out Cortex cluster using Docker. We'll need:
> - A built Cortex image.

This is why the example worked for whoever wrote it. It's go developer focused. If you run `make`, you get a cortex image.
But I agree is not clear and you should definitely specify a version. (v1.13.0 is fine)

The other thing that I would love to remove from this "getting started" is the notion of consul, I think we can just use memberlist. But we can leave that for another PR

File: docs/getting-started/_index.md [hunk 1]
```diff
@@ -78,17 +78,17 @@ Next we'll run a couple of Cortex instances pointed at that Consul.  You'll note
 
 ```sh
 $ docker run -d --name=cortex1 --network=cortex \
-    -v $(pwd)/docs/configuration/single-process-config-blocks.yaml:/etc/single-process-config-blocks.yaml \
+    -v $(pwd)/docs/configuration/single-process-config-blocks-local.yaml:/etc/single-process-config-blocks-local.yaml \
     -p 9001:9009 \
-    quay.io/cortexproject/cortex \
-    -config.file=/etc/single-process-config-blocks.yaml \
+    quay.io/cortexproject/cortex  \
```
Comment (friedrichg) [inline docs/getting-started/_index.md:83]:
```suggestion
    quay.io/cortexproject/cortex \
```

**Review by friedrichg — APPROVED** [review]

File: docs/getting-started/_index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ You can reconfigure the config to use GCS, Azure storage or local storage as sho
 
 ```sh
 $ go build ./cmd/cortex
-$ ./cortex -config.file=./docs/configuration/single-process-config-blocks.yaml
+$ ./cortex -config.file=./docs/configuration/single-process-config-blocks-local.yaml
 ```
 
 Unless reconfigured this starts a single Cortex node storing blocks to S3 in bucket `cortex`.
```
Comment (friedrich-at-adobe) [inline docs/getting-started/_index.md:30]: This comment no longer makes sense. Needs update

**Review by friedrich-at-adobe — CHANGES_REQUESTED** [review]

File: docs/getting-started/_index.md (hunk 2 above)
Comment (friedrichg) [inline docs/getting-started/_index.md:30]: this comment no longer makes sense. Needs update

**Review by friedrichg — CHANGES_REQUESTED** [review]

**Review by friedrichg — APPROVED** [review]

**Comment by alvinlin123** [comment]
> Umm, yeah I don't see we have a image tagged with `latest` on quay... So probably the doc doesn't work from the beginning.

We should fix this: https://github.com/cortexproject/cortex/issues/4951

**Review by alvinlin123 — APPROVED** [review]

**Pull request merged**